raspir/raspir.py

```diff
@@ -246,9 +255,19 @@ def make_freq_images(x, set_images):
         x_bio += [0]*b
         x_reference3 = np.sqrt(x_reference)
         sep = '_'
-        stripped_name = species_name.split(sep)
-        stripped_name2 = stripped_name[3] + ' ' + stripped_name[4]
-        stripped_name3 = stripped_name[3] + '_' + stripped_name[4]
+        # add error handling in case separator not present for some taxa. eg chrY etc from mouse
+        if sep in species_name:
+            try:
+                stripped_name = species_name.split(sep)
+                stripped_name2 = stripped_name[3] + ' ' + stripped_name[4]
+                stripped_name3 = stripped_name[3] + '_' + stripped_name[4]
+            except:
+                logging.info('Warning  Name could not be parsed correctly using "_" splits: {}', str(species_name))
+                stripped_name = species_name
+                stripped_name3 = species_name
+        else:
+            stripped_name = species_name
+            stripped_name3 = species_name
 
         fig, ax1 = plt.subplots(1, 1, figsize=(2.5, 2))
         fig.suptitle(stripped_name2, style='italic', fontsize=4)
```

Comment:
This line needs stripped_name2, but in the else part of the if statement above, as well as in the except part of try and except only stripped_name and stripped_name3 are declared. Could this be problematic?

Reply:
You're right, it does not look complete. I think I wrote this because chrY from mouse (no separators, neither " " nor "_" are present.

I think in the except block, it just gives up trying to parse any form of stripped_name out, and keeps the variable from species_name.  

stripped_name2 was not attributed, since it is not needed elsewhere in the code.

Reply:
I added some comments in the commits below, is it better now ?

Reply:
Isn't stripped_name2 used in line 276 ` fig.suptitle(stripped_name2, style='italic', fontsize=4)`

Reply:
Stimmt, wie habe ich das übersehen. So, das sollte auch mit den Mausdaten von damal getestet werden, ich suche ein Test raus